docs/device-management/5-ota/3-managing-cohorts.md

```diff
@@ -0,0 +1,57 @@
+---
+id: managing-cohorts
+title: Managing Cohorts
+---
+
+In order to deploy OTA updates to your devices, you'll need to assign them to a
+Cohort. A Cohort is a group of devices that should have the same set of
+packages, and receive the same firmware updates.
+
+Typically, you'll need to create a Cohort for each type of device in your
+project. For example, if your project contains both light switches and light
+bulbs, you should form one "Light Switch" cohort and one "Light Bulb" cohort, so
+that you can manage the firmware of the two types of devices separately. You may
+also want to create dedicated cohorts for devices that should receive the
+updates early, such as internal test devices or external beta testers.
+
+## Assigning devices to a Cohort
+
+There are three ways to assign a device to a Cohort in the Golioth Web Console:
+
+- In the Cohort's page:
+    1. Click the `Add Devices` button in the top right corner
+    2. Find the device you want to add to the Cohort in the table
+    3. Click the `Add` button next to your device in the table
+- In the Device Index:
+    1. Select the devices you want to add using the checkboxes in the device
+       list
+    2. Click `Bulk Actions` in the top right corner
+    3. Select `Assign to Cohort`
+    4. Select the right Cohort and press `Assign`
+- In the Edit Device page:
+    1. Click the `Edit` button in the top right corner
+    2. Select a Cohort in the `Cohort` dropdown in the form
+    3. Click `Save`
+
+If your device is online and observing the OTA manifest, it will immediately
+receive a manifest update and start updating its packages to match the Cohort's
+active deployment.
+
+
+## Removing devices from a Cohort
+
+You can remove a device from a cohort through the Device Index and the Edit
```

Comment:
nitpick: `Cohort` is used throughout this document, then we switch to using `cohort` here